include/seastar/core/reactor.hh

```diff
@@ -532,7 +532,13 @@ public:
     future<> chmod(std::string_view name, file_permissions permissions) noexcept;
 
     future<int> inotify_add_watch(int fd, std::string_view path, uint32_t flags);
-    
+
+    future<std::tuple<pid_t, int, int>>
+    spawn(std::string_view pathname,
```

Comment:
Is there any specific reason why this method should sit on reactor class?

Reply:
because it needs the access to the `reactor::_thread_pool` for performing the blocking syscalls.

Reply:
Please add a helper in seastar.hh so people don't need to include reactor.hh.

Reply:
i forgot to address this one. will do so tomorrow.

Reply:
added.